Gruntfile.js

```diff
@@ -23,8 +23,24 @@ module.exports = function (grunt) {
   var appConfig = {
     app: require('./bower.json').appPath || 'app',
     dist: 'client/dist'
+  }
+    // Configurable paths for the application
+  var bowerJson = require('./bower.json');
+  var bowerRc;
+  try {
+      bowerRc = JSON.parse(fs.readFileSync('./.bowerrc'), 'utf-8');
+  } catch(err) {
+      // No .bowerrc.  Not a problem--it's optional!
+  }
+  var appConfig = {
+      app: (bowerJson && bowerJson.appPath) ? bowerJson.appPath : 'client/ngapp',
+      // vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'client/ngapp/vendor',
+      vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'bower_components',
+      dist: 'client/dist',
+      stage: '.tmp'
```

Comment:
While `.bowerrc` is optional in general, this particular project does provide it. Can you please simplify this Grunt code to assume that `.bowerrc` is present and has both `appPath` and `directory` properties set?